Orange/classification/tree.py

```diff
@@ -64,6 +64,8 @@ def __init__(
         self.min_samples_split = min_samples_split
         self.sufficient_majority = sufficient_majority
         self.max_depth = max_depth
+        self.params = {k: v for k, v in vars().items()
+                       if k not in ('args', 'kwargs')}
```

Comment:
What is in `vars()` -- except globals, which we don't like anyway?! I don't like this blind copying.